### ISSUE TITLE
Request: "Are you sure you want to run task" confirmation before clicking Run button

### ISSUE THREAD

**Issue by ioogithub** [kapitainsky/RcloneBrowser#144]
It would be nice to have a dialoge box displayed that asks:

Are you sure you want to run 'Task'?

Since running a task with sync and other options can result in data loss, it is very easy to accidentally click the run button or to click the run when you meant to click Dry Run and experience data loss. This simple reminder could prevent this.

Ideally this simple confirmation could be configurable from the settings so it doesn't get in the way of users that don't want it.

**Comment by dia3olik** [comment]
I totally agree with this option to be included in preferences, great idea to safeguard data.

**Comment by MacAppSuggestions** [comment]
only optionally, not default please
